.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -294,11 +294,11 @@ jobs:
         run: |
           PREVIOUS_VERSION=$(cat oss/airbyte-connector-builder-resources/CDK_VERSION)
           sed -i "s/${PREVIOUS_VERSION}/${{needs.bump-version.outputs.new_cdk_version}}/g" oss/airbyte-connector-atelier-server/Dockerfile
+          sed -i "s/airbyte-cdk==${PREVIOUS_VERSION}/airbyte-cdk==${{needs.bump-version.outputs.new_cdk_version}}/g" oss/airbyte-connector-atelier-server/requirements.in
           echo ${{needs.bump-version.outputs.new_cdk_version}} > oss/airbyte-connector-builder-resources/CDK_VERSION
-          cd oss/airbyte-connector-builder-server
+          cd oss/airbyte-connector-atelier-server
```

Comment:
Just to confirm, this was the issue where we were testing the old python image and so the issue with the java server slipped unnoticed?

For the moment I agree we should fix this, but before we merge this, let's be extra careful to check that it hasn't been renamed because we've got a handful of workstreams swapping the builder for atelier

Reply:
This is a very good point. I'll be extra careful when I'll merge master into this branch. Thanks for reminding me!